README.md

```diff
@@ -8,6 +8,7 @@ The name of the script was inspired by the song «[Vote or Die!](http://southpar
 
 * Emojis-based.
 * The bot prepares in advance all the necessary emojis (`:0:`, `:1:`, `:2:`, etc), putting them under the polls to simplify searching for them to vote.
```

Comment:
It seems that instead of 
`(`:0:`, `:1:`, `:2:`, etc)`
there should be actual emodji codes
(`:zero:`, `:one:`, `:two:`, etc)